src/modules/auth/auth.service.ts

```diff
@@ -70,8 +69,8 @@ export class AuthService {
       .lowercase() // Must have lowercase letters
       .has()
       .digits() // Must have at least 2 digits
-      .has()
-      .symbols() // Must have symbols
+      // .has()
```

Comment:
I prefer remove that lines instead comment them what do you think? , it doesn't affect the functionality but I prefer don't send comment code to production.

Reply:
I think comments as documentation or reference are great. We should focus on removing un used imports and unnecessary commented code

Reply:
Oks. Already